### ISSUE TITLE
Help ensure fluid type meets WCAG criteria

### ISSUE THREAD

**Issue by mirisuzanne** [trys/utopia-core#1]
Hi, I love the utopia project - but there's always a risk with fluid type that [it will fail the WCAG criteria for zoom](https://adrianroselli.com/2019/12/responsive-type-and-zoom.html). Lately I've been checking the output of Utopia against the checker at https://fluid.style/ - but it seems like a single tool could handle both aspects of this math? Would there be any interest in adding an a11y checker to Utopia?

**Comment by trys** [comment]
@mirisuzanne that's a wonderful suggestion. Exposing it as a warning on this library would be a great start, but it would be super-nice to add it to [utopia.fyi](https://utopia.fyi) directly. I'll give it some thought!

**Comment by trys** [comment]
@mirisuzanne `utopia-core` now returns `wcagViolations` with the offending `from` and `to` viewports, and that's been piped into the [type calculator](https://utopia.fyi/type/calculator/?c=320,16,1.2,1240,20,1.414,5,2,&s=0.75%7C0.5%7C0.25,1.5%7C2%7C3%7C4%7C6,s-l&g=s,l,xl,12) to show any steps that don't meet SC 1.4.4.

Huge thanks to @barvian for providing the calculations from [fluid.style](https://fluid.style).

<img width="1552" alt="Screenshot 2024-01-29 at 19 27 49" src="https://github.com/trys/utopia-core/assets/3533780/e9e0a652-e5c4-447b-99c3-e5cb7d6bd3a1">

**Comment by mirisuzanne** [comment]
Yay, thank you! So quickly!
